flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/PerJobCheckpointRecoveryFactory.java

```diff
@@ -29,13 +30,13 @@
  * Simple {@link CheckpointRecoveryFactory} which creates a
  * {@link CompletedCheckpointStore} and a {@link CheckpointIDCounter} per {@link JobID}.
  */
-public class TestingCheckpointRecoveryFactory implements CheckpointRecoveryFactory {
+public class PerJobCheckpointRecoveryFactory implements CheckpointRecoveryFactory {
```

Comment:
Why did you call it `PerJobCheckpointRecoveryFactory`? Is it because it stores for different jobs the services? To me it sounds that this factory only works for a single job.

Reply:
yes, would `MockCheckpointRecoveryFactory` be more clear?